bin/mozilla-en-US.py

```diff
@@ -115,10 +115,18 @@ def push(path):
 dname = os.path.dirname(abspath)
 os.chdir(dname)
 
-for channel in ['aurora', 'beta']:
+for channel in ['beta', 'central']:
     for repo in SOURCE_REPOS:
         ending = repo + '-' + channel
-        url = 'ssh://hg.mozilla.org/releases/' + ending
+
+        url_folder = ''
+        if channel == 'central':
```

Comment:
Ah, I completely missed the following lines of code where you add l10n/ to the url_folder for central

Since there are only two repos, I wonder if you need this much code instead of a simple mapping? Unless I'm missing other usages of this code

```python
urls = {
    'central': 'ssh://hg.mozilla.org/l10n/mozilla-central',
}

repository = urls.get(channel, 'ssh://hg.mozilla.org/releases/mozilla-{0}'.format(channel))

```